CoroutineTemplate/data/src/main/java/co/nimblehq/coroutine/repositoryimpl/UserRepository.kt

```diff
@@ -0,0 +1,21 @@
+package co.nimblehq.coroutine.repositoryimpl
+
+import co.nimblehq.coroutine.data.service.ApiService
+import co.nimblehq.coroutine.data.service.response.toUserEntities
+import co.nimblehq.coroutine.entity.UserEntity
+import co.nimblehq.coroutine.repository.UserRepository
+import co.nimblehq.coroutine.usecase.UseCaseResult
+
+class UserRepositoryImpl constructor(
+    private val apiService: ApiService,
+) : UserRepository {
+
+    override suspend fun getUsers(page: Int, size: Int): UseCaseResult<List<UserEntity>> {
```

Comment:
why would we need to pass the pagination to get a User profile? 🤔  especially that it's not being used in the later call:

`val response = apiService.getUsers()`

Reply:
@sleepylee Yes, they must be removed as redundant.